src/Build.UnitTests/BackEnd/TaskHostConfiguration_Tests.cs

```diff
@@ -336,7 +336,7 @@ public void TestTranslationWithEmptyDictionary()
             TaskHostConfiguration deserializedConfig = packet as TaskHostConfiguration;
 
             Assert.Equal(config.TaskName, deserializedConfig.TaskName);
-#if FEATURE_ASSEMBLY_LOADFROM
+#if !FEATURE_ASSEMBLYLOADCONTEXT
             Assert.Equal(config.TaskLocation, config.TaskLocation);
```

Comment:
These look fishy: separate PR to make it

```suggestion
            Assert.Equal(config.TaskLocation, deserializedConfig.TaskLocation);
```
?

Reply:
Done in #4297